.docker/README.md

```diff
@@ -0,0 +1,78 @@
+# HarbourBridge: Docker Mode
```

Comment:
`.<name>` convention is generally used for hidden and  other unrelated folders, why is it `.docker` and not `docker`?

Reply:
`.docker` has only the configuration file for the automated release process of the docker images. Configuration files are generally put in hidden folders.